packages/mcp/src/server/tools/list-vibe-icons.ts

```diff
@@ -0,0 +1,50 @@
+import { getErrorMessage, MCPTool } from "../index.js";
+import { IconMetadataService } from "./icon-metadata-service.js";
+
+export const listVibeIconsTool: MCPTool<{}> = {
+  name: "list-vibe-icons",
+  description:
+    "Get a list of all available Vibe icons from the @vibe/icons package. Returns icon names, descriptions, categories, and tags.",
+  inputSchema: {},
+  execute: async () => {
+    try {
+      const allIcons = await IconMetadataService.getIconMetadata();
+
+      // Format the response to include relevant information
+      const formattedIcons = allIcons.map(icon => ({
+        name: icon.name,
+        description: icon.description,
+        category: icon.category || [],
+        tags: icon.tags.split(", ").map(tag => tag.trim())
+      }));
```

Comment:
you can combine get-vibe-icon-details with list-vibe-icons if you're planning to return all the metadata in the "list".
you can add an optional param here of icon name - so if a user doesn't supply an icon name it would receive everything, otherwise receive specific supplied name